jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/signin.js

```diff
@@ -211,4 +212,28 @@ function sendLoginPost(e) {
             });
         }
     });
-};
\ No newline at end of file
+}
+
+function isPreventSubmitFor($element) {
+    var isPreventFormSubmit = false;
+
+    $.each(jDialog.options.preventSubmitElements, function (idx, value) {
```

Comment:
Could we unwrap it into usual JS iteration (with `for(){}`)? It will be 1 additional line, but we'll unbind ourselves from JQuery (on the off chance we'll be migrating from it or from its current version). 

There is also a similar native function: [Array.prototype.forEach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) which does a similar thing. Won't work in IE8 though.